onnxruntime/test/unittest_util/checkers.cc

```diff
@@ -225,17 +225,27 @@ template <>
 struct TensorCheck<Int4x2> {
   void operator()(const Tensor& expected, const Tensor& actual, const ValidateOutputParams& params,
                   const std::string& /*provider_type*/) const {
-    ORT_UNUSED_PARAMETER(params);
+    const bool has_abs_err = params.absolute_error.has_value();
+    Tensor expected_sorted, actual_sorted;
     const Int4x2* cur_expected;
     const Int4x2* cur_actual;
     const auto size = narrow<size_t>(actual.Shape().Size());
     cur_expected = expected.Data<Int4x2>();
     cur_actual = actual.Data<Int4x2>();
+    double threshold = 0.0f;
```

Comment:
>double threshold = 0.0f; [](http://example.com/codeflow?start=4&length=24)

Since there is an if below you can use it w/o additional var

Reply:
Reverted